extensions/ql-vscode/src/model-editor/model-editor-view.ts

```diff
@@ -474,9 +473,16 @@ export class ModelEditorView extends AbstractWebview<
 
   private async generateModeledMethodsFromLlm(
     packageName: string,
-    methods: Method[],
-    modeledMethods: Record<string, ModeledMethod>,
+    methodSignatures: string[],
   ): Promise<void> {
+    const methods = this.modelingStore.getMethods(
```

Comment:
We could also have a function in the modeling store that returns a tuple instead of calling the modeling store twice.

Reply:
Yeah agree! I said the same thing [earlier](https://github.com/github/vscode-codeql/pull/2929#discussion_r1351770667) actually 😄